### PR TITLE
config object tests

### PR DESCRIPTION
**Pull request by kevinburleigh75**

### CODE REVIEW

File: app/src/config.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    @staticmethod
+    def get_configs_from_dynamodb(region_name, table_name):
+        session = boto3.session.Session(region_name=region_name)
+        ddb_client = session.client('dynamodb')
+
+        try:
+            ## Fetch all configs with is_current set to true.
+            ## NOTE: This assumes that DynamoDb does not
+            ##       process more than 1MB of data to find
+            ##       the desired items.
```
Comment (jpslav) [inline app/src/config.py:57]: Likelihood that this is true?

File: app/tests/unit/config/test_config_data_usage.py [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+def test_uri_map_missing_pretty_uri():
+    with mock.patch('src.config.Config.get_configs_from_dynamodb') as mock_dynamo:
+        mock_dynamo.return_value = [
+            load_config(
+                config_name = 'config_template',
+                access_map  = json.dumps({}),
+                version     = json.dumps('ver_42'),
+                uri_map     = json.dumps({
+                    '/some-pretty-path/pretty-resource': {
+                        'S': '/some-ugly-path/ugly-resource'
```
Comment (jpslav) [inline app/tests/unit/config/test_config_data_usage.py:62]: Think calls to load config can be dry’ed up? Eg make ‘config_template’ and {} be defaults and take dictionaries and have load config call json.dumps?

**Review by jpslav — APPROVED** [review]

File: app/src/config.py (hunk 1 above)
Comment (kevinburleigh75) [reply app/src/config.py:57]: Very high unless we leave tons and tons of ancient configs lingering in the database.

File: app/tests/unit/config/test_config_data_usage.py (hunk 2 above)
Comment (kevinburleigh75) [reply app/tests/unit/config/test_config_data_usage.py:62]: Done.

**Pull request merged**